### PR TITLE
[15.0][MIG] contract_payment_mode

### PR DESCRIPTION
**Pull request by palkesh-aktivsoftware**
Standard Migration

### CODE REVIEW

**Review by misern2 — APPROVED** [review]
LGTM

**Review by eduardgm — APPROVED** [review]
Functional review, LGTM!

File: contract_payment_mode/hooks.py [hunk 1]
```diff
@@ -10,14 +10,14 @@
...
-        m_contract = env["contract.contract"]
-        contracts = m_contract.search([("payment_mode_id", "=", False)])
-        if contracts:
-            _logger.info("Setting payment mode: %d contracts" % len(contracts))
-        for contract in contracts:
-            payment_mode = contract.partner_id.customer_payment_mode_id
-            if payment_mode:
-                contract.payment_mode_id = payment_mode.id
-        _logger.info("Setting payment mode: Done")
+    # with api.Environment.manage():
```
Comment (rousseldenis) [inline contract_payment_mode/hooks.py]: Why do you remove the context manager  ?

Comment (rousseldenis) [reply contract_payment_mode/hooks.py]: Ok, deprecated.

@palkesh-aktivsoftware Could you just remove the unneeded comment ?

Comment (palkesh-aktivsoftware) [reply contract_payment_mode/hooks.py]: > 

Sure, I made this changes. Please review it.

**Comment by marylla** [comment]
Hi @rousseldenis Can you review again and merge?

**Review by rousseldenis — APPROVED** [review]

**Review by marcelsavegnago — APPROVED** [review]

**Comment by marylla** [comment]
@pedrobaeza @rousseldenis Who wants to merge this? It has a lot of approvals now. :)

**Pull request merged**